apps/extension/src/storage/VaultStorage.ts

```diff
@@ -12,7 +12,7 @@ const Uint8ArrayCodec = new t.Type<Uint8Array, Uint8Array, unknown>(
     try {
       const a = Array.from(u as Uint8Array);
       return t.success(Uint8Array.from(a));
-    } catch (e) {
+    } catch (_e) {
```

Comment:
Could we just leave this empty?

Reply:
Will do that for other occurrences as well